physionet-django/user/templatetags/user_templatetags.py

```diff
@@ -7,6 +8,8 @@
 @register.inclusion_tag('user/settings_tabs.html')
 def settings_tabs(hide_password_settings: bool):
     default_tabs = ['Profile', 'Emails', 'Username', 'Cloud', 'ORCID', 'Credentialing', 'Training', 'Agreements']
+    if settings.ENABLE_RESEARCH_ENVIRONMENTS:
+        default_tabs.remove('Cloud')
```

Comment:
I don't think this logic is correct - having research environments enabled does not necessarily imply that we won't have Cloud integrations. This looks more like a completely new setting.

Maybe in the future we should align the `CloudInformation` model with the `CloudIdentity` model?